test/bats/azure.bats

```diff
@@ -55,6 +55,7 @@ setup() {
 }
 
 @test "CSI inline volume test" {
+  skip
```

Comment:
If we want to skip these tests, should we just remove them?

Reply:
Makes sense! Removed it.